bundle/src/main/java/com/adobe/acs/commons/redirects/servlets/RewriteMapServlet.java

```diff
@@ -62,16 +62,16 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
         }
         Collection<RedirectRule> rules = RedirectFilter.getRules(request.getResource());
         PrintWriter out = response.getWriter();
-        out.printf("# %s Redirects\n", statusCode == 0 ? "All" : "" + statusCode);
+        out.printf("# %s Redirects%n", statusCode == 0 ? "All" : "" + statusCode);
```

Comment:
@YegorKozlov after starting to use %n as line separator, splitting only by \n in tests by
```
String[] lines = response.getOutputAsString().split("\n");
```
in https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/3484/commits/858fda63ef67e26ec8d0fa5f6424a2e94557fc19#diff-36adc0c81c9585f4cadf9e45117faabfdda5ef6d7e197ade7d39e0a896c6853eR87
might not be precise enough anymore on Windows